Java/TelemetrySystem/src/main/java/tddmicroexercises/telemetrysystem/TelemetryDiagnosticInterface.java

```diff
@@ -0,0 +1,13 @@
+package tddmicroexercises.telemetrysystem;
+
+public interface TelemetryDiagnosticInterface {
+
+    public void connect(String msg);
+
+    public void disconnect();
+
+
+    void checkTransmission() throws Exception;
+
+    Object getDiagnosticInfo();
```

Comment:
we can have connect and disconnect in a different interface
As in here, we need to update our code when ever we need to make changes on connection code & for check transmission

Reply:
applied in new commit